bin/docker-push

```diff
@@ -17,6 +20,11 @@ if [ -z "$CI_REF" ]; then echo CI_BRANCH or CI_TAG must be set; exit 1; fi
 
 CI_REF=$(echo "${CI_REF}" | tr / _)
 
+# support list of additional tag versions delimited by commas by default
+TAG_VERSION_DELIMITER_CHAR="${TAG_VERSION_DELIMITER_CHAR:-,}"
```

Comment:
Could this also be `DOCKER_TAG_VERSION_DELIMITER_CHAR` to specify what type of tag this is used for in the pipeline?

Reply:
sure